_vagrant/salt/pillar/README.md

```diff
@@ -0,0 +1 @@
+A complete sample of the pillar tree should be added here, as promised.
```

Comment:
As above, we can remove this file as GitBook will never process it (being outside of the `_docs` directory). We'll either remove the corresponding link or point it directly at GitHub so we're able to show the directory listing.